proxy/envoy/config.go

```diff
@@ -642,7 +642,7 @@ func buildInboundListeners(mesh *proxyconfig.MeshConfig, sidecar proxy.Node,
 						// Note: websocket routes do not call the filter chain. Will be
 						// resolved in future.
 						if mesh.MixerAddress != "" {
-							websocketRoute.OpaqueConfig = buildMixerOpaqueConfig(true, false)
+							websocketRoute.OpaqueConfig = buildMixerOpaqueConfig(!mesh.DisablePolicyChecks, false)
```

Comment:
Has any testing been done with websocket ? which metrics can be collected?

Reply:
None that I'm aware of. It's still HTTP/1.1.

Reply:
Mixer filter will not work with Websockets. Envoy does not activate the filter chain when websocket is enabled. Work in progress.